view/src/main/java/cmm/apps/esmorga/view/eventdetails/EventDetailsView.kt

```diff
@@ -44,7 +43,7 @@ import org.koin.core.parameter.parametersOf
 
 @Screen
 @Composable
-fun EventDetailsScreen(eventId: String, onBackPressed: () -> Unit, edvm: EventDetailsViewModel = koinViewModel(parameters = { parametersOf(eventId) })) {
+fun EventDetailsScreen(eventId: String, onBackPressed: () -> Unit, edvm: EventDetailsViewModel = koinViewModel(parameters = { parametersOf(eventId) }), onPrimaryButtonClicked: () -> Unit) {
```

Comment:
I would not call it `onPrimaryButtonClicked` because in current design primary button can be login, but also join/leave event.